app/assets/stylesheets/application.css

```diff
@@ -15,3 +15,37 @@
  *= require foundation_and_overrides
 
  */
+
+
+body {
+  background-color: black;
+}
+
+.tile {
+  margin: 20px;
+  padding: 8px 30px;
+  border: 4px;
+  border-radius: 10px;
+  height: 400px;
+  width: 280px;
+  border-style: solid;
+  border-color: gray;
+  background-color: #c0fefe;
+  text-align: center;
+  font-family: helvetica;
```

Comment:
generally it's good to set a cascading list of fonts from most specific to most generic. I would suggest adding a sans-serif mention here